src/Request.php

```diff
@@ -49,7 +49,10 @@ private static function clone($value)
     private $tlsHandshakeTimeout = 10000;
 
     /** @var int */
-    private $transferTimeout = 10000;
+    private $transferTimeout = 60000;
```

Comment:
I don't see any reason to change the default here.

Reply:
I was thinking for better UX when requesting a large file. Having all the timeouts the same isn't really helpful.

Reply:
I think downloading large files is rather uncommon compared to making API calls where large timeouts aren't a good idea.

Reply:
Already reverted the timeouts back to pre-PR values.